website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -86,53 +86,85 @@ HCP Terraform prioritizes and overwrites conflicting variables according to the
 
```

Comment:
Right above here in the precedence section can you add information about how the entity owning the variable set affects precedence? With an example too please!